File: app/src/main/java/com/dicoding/moviecataloguerv/data/source/remote/response/CreditsResponse.java
package com.dicoding.moviecataloguerv.data.source.remote.response;

import com.dicoding.moviecataloguerv.data.source.model.Cast;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import lombok.Data;

@Data
public class CreditsResponse {
    @SerializedName("id")
    @Expose
    private int id;

    @SerializedName("cast")
    @Expose
    private List<Cast> cast;
}

File: core/src/main/kotlin/com/ryanrvldo/core/data/source/remote/response/commons/CreditsResponse.kt
package com.ryanrvldo.core.data.source.remote.response.commons

import com.google.gson.annotations.SerializedName

data class CreditsResponse(
    @SerializedName("id") val id: Int?,
    @SerializedName("cast") val casts: List<CreditResponse>?,
    @SerializedName("crew") val crews: List<CreditsResponse>?,
)
